src/dependency-manager/utils/errors.ts

```diff
@@ -96,5 +97,6 @@ export class ValidationErrors extends ArchitectError {
 
     this.message = JSON.stringify(errors, null, 2);
     this.file = file;
+    this.exitCode = exitCode || 1;
```

Comment:
do we actually need to set this exitCode? I would assume the super.catch would always default to exitCode=1

Reply:
I'll remove the exitCode property.  It will default to exitCode = 1 by calling the super.catch() even without the exitCode set.  The initial thought was to make it more flexible by allowing for other exitCodes if specified.

Reply:
reverted errors.ts